client/src/partials/price_list/pricelist.html

```diff
@@ -41,40 +42,41 @@
                 <td><a class="btn btn-xs btn-default" id="price_delete_{{ $index+1 }}" ng-click="PriceListCtrl.del(list)" ><i class="glyphicon glyphicon-trash"></i></a></td>
               </tr>
               <tr ng-if="PriceListCtrl.priceList.data.length===0">
-                <td colspan="3"><i>{{ 'PRICE_LIST.NO_RECORDS' | translate }}</i></tr>
+                <td colspan="3"><i translate> PRICE_LIST.NO_RECORDS </i></tr>
               </tr>
             </tbody>
           </table>
+
         </div>
       </div>
 
-      <div class="col-xs-6" ng-switch="PriceListCtrl.view">
+      <div class="col-md-6" ng-switch="PriceListCtrl.view">
         <div ng-switch-default>
           <div class="alert alert-info" id="default">
-            <h4>{{ "PRICE_LIST.TITLE" | translate }}</h4>
-            <p>{{ "PRICE_LIST.HELP_TXT_1" | translate }}</p>
+            <h4 translate> PRICE_LIST.TITLE </h4>
+            <p translate> PRICE_LIST.HELP_TXT_1 </p>
             <ul style="list-style-type: none; padding-left: 15px;">
-              <li><i class="glyphicon glyphicon-pencil"></i> {{ "PRICE_LIST.HELP_TXT_2" | translate }}</li>
-              <li><i class="glyphicon glyphicon-trash"></i> {{ "PRICE_LIST.HELP_TXT_3" | translate }}</li>
+              <li><i class="glyphicon glyphicon-pencil"></i><span translate> PRICE_LIST.HELP_TXT_2 </span> </li>
```

Comment:
In the future, we should swap these glyphicons out with font awesome.  That can be another PR though.